lib/ui/view/dm_user/dm_user_view.dart

```diff
@@ -61,11 +63,11 @@ class DmUserView extends StatelessWidget with $DmUserView {
           body: Stack(
             children: [
               ExpandableTextFieldScreen(
-                channelID: '',
+                channelID: 'DM',
```

Comment:
Please add your Strings to the strings file and reference it here

Reply:
Ok, noted